### PR TITLE
Cleanup and update commons-lang #7703

### PR DESCRIPTION
**Pull request by rymsha**

### CODE REVIEW

**Comment by jsi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           

Complexity increasing per file
==============================
- modules/core/core-api/src/main/java/com/enonic/xp/page/DescriptorKey.java  1
- modules/lib/lib-node/src/main/java/com/enonic/xp/lib/node/QueryAggregationParams.java  4
- modules/core/core-api/src/main/java/com/enonic/xp/macro/MacroKey.java  1
- modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/page/region/CreateFragmentCommand.java  2
- modules/lib/lib-content/src/main/java/com/enonic/xp/lib/content/QueryAggregationParams.java  4
- modules/core/core-api/src/main/java/com/enonic/xp/site/mapping/ContentMappingConstraint.java  5
- modules/core/core-api/src/main/java/com/enonic/xp/schema/BaseSchemaName.java  1
         

Clones added
============
- modules/core/core-api/src/main/java/com/enonic/xp/security/auth/VerifiedUsernameAuthToken.java  1
- modules/core/core-api/src/main/java/com/enonic/xp/security/auth/UsernamePasswordAuthToken.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/enonic/xp/pullRequest?prid=4635733&bid=15533631)

File: modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/ContentAuditLogSupport.java [hunk 1]
```diff
@@ -42,346 +50,531 @@
...
 class ContentAuditLogSupport
 {
-
     private static final String SOURCE_CORE_CONTENT = "com.enonic.xp.core-content";
 
+    private static final int THREAD_POOL_SIZE = Runtime.getRuntime().availableProcessors();
+
+    private static final int QUEUE_SIZE = 100;
+
+    private static final ThreadPoolExecutor executor =
```
Comment (jsi) [inline modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/ContentAuditLogSupport.java]: Codacy Issue found: [Name 'executor' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.](https://app.codacy.com/app/enonic/xp/pullRequest?prid=4635733)

**Comment by jsi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- modules/web/web-header/src/test/java/com/enonic/xp/web/impl/header/HeaderFilterWrapperTest.java  1
- modules/lib/lib-content/src/main/java/com/enonic/xp/lib/content/QueryAggregationParams.java  4
- modules/core/core-api/src/main/java/com/enonic/xp/schema/BaseSchemaName.java  1
- modules/core/core-api/src/main/java/com/enonic/xp/audit/LogAuditLogParams.java  1
- modules/server/server-rest/src/main/java/com/enonic/xp/impl/server/rest/task/ReprocessRunnableTask.java  6
- modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/elasticsearch/snapshot/RestoreResultFactory.java  1
- modules/lib/lib-node/src/main/java/com/enonic/xp/lib/node/QueryAggregationParams.java  4
- modules/lib/lib-node/src/main/java/com/enonic/xp/lib/node/mapper/AbstractQueryResultMapper.java  3
- modules/web/web-header/src/main/java/com/enonic/xp/web/impl/header/HeaderFilterWrapper.java  1
- modules/server/server-rest/src/main/java/com/enonic/xp/impl/server/rest/model/ReprocessContentResultJson.java  2
- modules/core/core-api/src/main/java/com/enonic/xp/site/mapping/ContentMappingConstraint.java  5
- modules/admin/admin-impl/src/main/java/com/enonic/xp/admin/impl/rest/resource/issue/IssueUpdatedMailMessageGenerator.java  1
- modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/page/region/CreateFragmentCommand.java  2
- modules/core/core-api/src/main/java/com/enonic/xp/macro/MacroKey.java  1
- modules/web/web-api/src/main/java/com/enonic/xp/web/filter/FilterConfigInitParametersOverride.java  1
- modules/lib/lib-content/src/main/java/com/enonic/xp/lib/content/mapper/PublishContentResultMapper.java  3
- modules/core/core-api/src/main/java/com/enonic/xp/page/DescriptorKey.java  1
         

Complexity decreasing per file
==============================
+ modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/ContentAuditLogSupport.java  -2
+ modules/lib/lib-node/src/main/java/com/enonic/xp/lib/node/mapper/NodeQueryResultMapper.java  -1
         

Clones added
============
- modules/core/core-api/src/main/java/com/enonic/xp/security/auth/UsernamePasswordAuthToken.java  1
- modules/core/core-api/src/main/java/com/enonic/xp/security/auth/VerifiedUsernameAuthToken.java  1
         

Clones removed
==============
+ modules/core/core-api/src/main/java/com/enonic/xp/audit/LogAuditLogParams.java  -1
+ modules/core/core-api/src/main/java/com/enonic/xp/audit/AuditLog.java  -1
+ modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/PublishContentCommand.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/enonic/xp/pullRequest?prid=4635733&bid=15533631)

**Comment by rymsha** [comment]
this one was started from master. Don't change it to next as it will mess up. Wait for next to be master again

**Comment by jsi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           

Complexity increasing per file
==============================
- modules/core/core-api/src/main/java/com/enonic/xp/site/mapping/ContentMappingConstraint.java  5
- modules/core/core-api/src/main/java/com/enonic/xp/macro/MacroKey.java  1
- modules/core/core-api/src/main/java/com/enonic/xp/schema/BaseSchemaName.java  1
- modules/script/script-impl/src/main/java/com/enonic/xp/script/impl/util/ScriptLogger.java  2
- modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/page/region/CreateFragmentCommand.java  2
- modules/lib/lib-content/src/main/java/com/enonic/xp/lib/content/QueryAggregationParams.java  4
- modules/lib/lib-node/src/main/java/com/enonic/xp/lib/node/QueryAggregationParams.java  4
- modules/core/core-api/src/main/java/com/enonic/xp/page/DescriptorKey.java  1
         

Clones added
============
- modules/core/core-api/src/main/java/com/enonic/xp/security/auth/VerifiedUsernameAuthToken.java  1
- modules/core/core-api/src/main/java/com/enonic/xp/security/auth/UsernamePasswordAuthToken.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/enonic/xp/pullRequest?prid=4635733&bid=15533631)

**Comment by GlennRicaud** [comment]
I let you fix the conflicts. Approved

**Comment by jsi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           

Complexity increasing per file
==============================
- modules/core/core-api/src/main/java/com/enonic/xp/macro/MacroKey.java  1
- modules/core/core-api/src/main/java/com/enonic/xp/site/mapping/ContentMappingConstraint.java  5
- modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/page/region/CreateFragmentCommand.java  2
- modules/core/core-api/src/main/java/com/enonic/xp/schema/BaseSchemaName.java  1
- modules/core/core-api/src/main/java/com/enonic/xp/page/DescriptorKey.java  1
- modules/lib/lib-node/src/main/java/com/enonic/xp/lib/node/QueryAggregationParams.java  4
- modules/lib/lib-content/src/main/java/com/enonic/xp/lib/content/QueryAggregationParams.java  4
         

Clones added
============
- modules/core/core-api/src/main/java/com/enonic/xp/security/auth/VerifiedUsernameAuthToken.java  1
- modules/core/core-api/src/main/java/com/enonic/xp/security/auth/UsernamePasswordAuthToken.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/enonic/xp/pullRequest?prid=4635733&bid=15533631)

**Pull request merged**